components/pokeartscreen.brs

```diff
@@ -1,8 +1,44 @@
 function init()
+    m.playButton = m.top.findNode("playVideoButton")
     m.ratingLabel = m.top.findNode("artScreenRatingLabel")
     m.pokemonSplashArtPoster = m.top.findNode("pokemonPoster") 
+    m.firstChild = 0
     m.top.observeField("contenturi","updatePoster")
     m.top.observeField("pokemonratinglabel", "updateRatingLabel")
+    getRequest("https://my-json-server.typicode.com/bogdanterzea/pokemon-server/videos")
+    observePlayButton()
```

Comment:
This is leading to a bug if the user moves too fast.
Observe the button just after the population of the data on it.